.travis.yml

```diff
@@ -13,6 +13,8 @@ addons:
       - postgresql-client
       - postgresql-server-dev-all
 
+  postgresql: "9.6"
```

Comment:
@bitemyapp I bumped PostgreSQL version. Hopefully, that solves the issue since I tested locally using the same version `9.6`.